Public/Src/Engine/Processes/UnSandboxedProcess.cs

```diff
@@ -393,7 +393,7 @@ protected void LogProcessState(string message)
             if (LoggingContext != null)
             {
                 string fullMessage = I($"Exited: {m_processExecutor?.ExitCompleted ?? false}, StdOut: {m_processExecutor?.StdOutCompleted ?? false}, StdErr: {m_processExecutor?.StdErrCompleted ?? false}, Reports: {ReportsCompleted()} :: {message}");
-                Tracing.Logger.Log.LogDetoursDebugMessage(LoggingContext, PipSemiStableHash, PipDescription, fullMessage);
+                Tracing.Logger.Log.LogDetoursDebugMessage(LoggingContext, PipSemiStableHash, string.Empty, fullMessage);

```

Comment:
>LogDetoursDebugMessage [](start = 35, length = 22)

how about removing the pipDescription parameter from the log event too?

Reply:
will do